README.md

```diff
@@ -149,11 +149,25 @@ This command is used to check one or more plugins against one or more IDEs ([opt
         [-team-city | -tc ]
         [-tc-grouping | -g ]
         [-external-prefixes <':'-separated list>]
+        [-suppress-internal-api-usages no|internal-plugins]
 
 `<plugins>` is either `<plugin path>` or `'@<file>'` with a list of plugin paths to verify, separated by a newline.
 
 `<IDE>` is either a path to local IDE installation, or an IDE pattern (see bellow in the [common options](#common-options)) 
 
+#### Specific options
+
+* `-suppress-internal-api-usages` will suppress internal API usages from internal plugins.
+This option is used by JetBrains MarketPlace by default. 
```

Comment:
JetBrains Market<p>lace

"internal plugins" is misleading, it should be "plugins from JetBrains" (see also below)

the cmd-line option should also be renamed